sound/soc/sof/Kconfig

```diff
@@ -68,6 +68,7 @@ config SND_SOC_SOF_NOCODEC
 
 config SND_SOC_SOF_NOCODEC_SUPPORT
 	bool "SOF nocodec mode support"
+	select SND_SOC_DMIC
```

Comment:
no, that's not necessary. You can use a dummy codec, we only use SND_SOC_DMIC to enable the use of DMIC with GPIOs as well as play with delays.

Reply:
OK, I will try to test without this.